lib/ACLMicroservicesBuilder.js

```diff
@@ -229,42 +226,39 @@ ACLMicroservicesBuilder.prototype._executeSavePermissions = function(args, callb
 
   if(args.ent.id) { // update
 
-    self.ACLMicroservicesBuilder._loadAndAuthorize(entityDef, args.ent.id, args.cmd, roles, context, args.showSoftDenied$, function(err, dbEntity) {
+    self.ACLMicroservicesBuilder._loadAndAuthorize(entityDef, args.ent.id, args.cmd, roles, context, args.showSoftDenied$,args.ent, function(err, dbEntity) {
```

Comment:
args.ent gets filtered based on the filter rules from the state of dbEntity which is loaded inside _loadAndAuthorize.